readme.md

```diff
@@ -539,6 +539,7 @@ Please refer run with `--help` for latest information.
 - writeImageXMPPerImage
 
   Set to `True` to export the image information to a .XMP sidecar file, one per image in the album. The data contained within the file is the same but some software requires matching file names to detect the metadata. If set to `True`, then `writeImageXMP` is ignored.
+  Additionally, enabling this option will create a .XMP sidecar for every ugoira encoding enabled, and allow you to customise the name of each file using `%image_ext%`. For example, if you enable `createWebp` and `createGif`, then set your `filenameInfoFormat` to something like `%urlFilename%.%image_ext%`, then you will end up with `<image ID>.gif.xmp` and `<image ID>.webp.xmp` files created.
```

Comment:
correct me if I'm wrong, but if `%image_ext%` not used in the filename format, when both ugoira and webm flag enabled for example, then it will only create 1 xmp file, right?

Reply:
> correct me if I'm wrong, but if `%image_ext%` not used in the filename format, when both ugoira and webm flag enabled for example, then it will only create 1 xmp file, right?
That's correct, it will try to create multiple ones but they will just be overwritten.